coos.html

```diff
@@ -460,43 +464,51 @@ <h3>Base GSBPM model</h3>
 
+				<p>The links between the different individuals are represented by the usual SKOS properties: all the sub-processes of a phase have a <code>skos:broader</code> link to the phase, and each phase has a <code>skos:broader</code> link to the <code>statisticalProductionProcess</code> individual.</p>
+				
 				<pre class="example nohighlight" title="Example of GSBPM phase and sub-process">
-				&lt;http://id.unece.org/activities/phase/2&gt; a coos:Phase ;
+				&lt;http://id.unece.org/activities/phase/2&gt; a coos:ActivityCategory ;
 					skos:notation "2" ;
 					skos:prefLabel "Design"@en ;
 					skos:narrower &lt;http://id.unece.org/activities/subProcess/2.3&gt; .
 
-				&lt;http://id.unece.org/activities/subProcess/2.3&gt; a coos:SubProcess ;
+				&lt;http://id.unece.org/activities/subProcess/2.3&gt; a coos:ActivityCategory ;
 					skos:notation "2.3" ;
 					skos:prefLabel "Design collection"@en ;
-					skos:broader &lt;http://id.unece.org/activities/phase/2&gt; ;
-					dcterms:isPartOf &lt;http://id.unece.org/activities/phase/2&gt; .
+					skos:broader &lt;http://id.unece.org/activities/phase/2&gt; .
 				</pre>
+				
+				<p>To materialize a clear distinction between the model itself and the statistical process which is modeled, the ontology defines classes that inherit from <code>prov:Activity</code>. Using PROV allows to describe an activity that really happens at a certain time, to specify who undertakes the activity and what entities it consumes or produces.</p>
+					
+				<p>Basically, three classes are defined in this GSBPM ontology: <code>StatisticalProductionActivity</code>, <code>Phase</code> and <code>SubProcess</code>.</p>
+
+				<p><code>StatisticalProductionActivity</code> is a sub-class of <code>prov:Activity</code>. Instances of this class are intended to be larger than a GSBPM phase, if they corresponds to a statistical production process a link to the <code>statisticalProductionProcess</code> individual is made using the property <code>dcterms:type</code>.</p>
+
+				<pre class="example nohighlight" title="Examples of statistical production activities">
+				ex:produce-study-2021 a coos:StatisticalProductionActivity ;
+					rdfs:label "Produce study on legal population 2021"@en .
+					
+				ex:produce-census-2021 a coos:StatisticalProductionActivity ;
+					rdfs:label "Produce census 2021"@en ;
+					dcterms:type coos:statisticalProductionProcess .
+				</pre>
+
+				<p><code>Phase</code> and <code>SubProcess</code> are both sub-classes of <code>StatisticalProductionActivity</code>. A link to the GSBPM taxonomy using the property <code>dcterms:type</code> is made on instances of these classes to formalize to what part of the GSBPM the activity that happened corresponds. Additionally, the idea that each phase is made up of its different sub-processes can be represented by using the Dublin Core dcterms:hasPart and dcterms:isPartOf properties.</p>
```

Comment:
Use <code> around dcterms:ashpart and dcterms:isPartOf.

Reply:
I don't understand this comment

Reply:
done